source/app/models/RockPaperScissors.rb

```diff
@@ -0,0 +1,26 @@
+class RockPaperScissors
+
+  def initialize(params)
+    @player_1_id = params['player_1_id'].to_i
+    @player_2_id = params['player_2_id'].to_i
+    @player_1_token_id = params['player_1_token_id'].to_i
+    @player_2_token_id = params['player_2_token_id'].to_i
+  end
+
+  def determine_winner
```

Comment:
You can make this logic a little stronger.... why so much if else /if going on here?